crates/solidity/inputs/schema/grammar/02-definitions/08-functions/productions.yml

```diff
@@ -22,18 +22,33 @@
 
 - name: "FunctionAttribute"
   kind: "Parser"
-  unversioned:
-    choice:
-      - reference: "ModifierInvocation"
-      - reference: "OverrideSpecifier"
-      - reference: "ExternalKeyword"
-      - reference: "InternalKeyword"
-      - reference: "PayableKeyword"
-      - reference: "PrivateKeyword"
-      - reference: "PublicKeyword"
-      - reference: "PureKeyword"
-      - reference: "ViewKeyword"
-      - reference: "VirtualKeyword"
+  versioned:
+    0.4.11:
+      choice:
+        - reference: "ModifierInvocation"
```

Comment:
nit: I suggest keeping these choices alphabetically sorted, to make it easier to compare/read.
I usually use the `Sort Lines Ascending` command in vscode.